### PR TITLE
Dashboard wordcloud + game count + tooltip improvements

### PR DESCRIPTION
**Pull request by paulauus**
Added genre selection to the wordcloud,
+ game count on the games page before table,
+ made tooltips look nice.

### CODE REVIEW

File: dashboard/functions.py [hunk 1]
```diff
@@ -110,6 +113,10 @@ def display_game_table(show_nsfw, os_selection, start_date, end_date, search_que
     elif sort_by == "Date (Newest)":
         table_data = table_data.sort_values("Release Date", ascending=False)
 
+    # Display the total number of games
+    total_games = len(table_data)
```
Comment (Leon-JS02) [inline dashboard/functions.py:117]: Nice

**Review by Leon-JS02 — APPROVED** [review]
This is a nice last minute change to the dashboard.

**Review by MichaelHassan — APPROVED** [review]
nice addition, adding more interactivity to dashboard.

File: dashboard/Dashboard.py [hunk 2]
```diff
@@ -98,9 +98,16 @@
     st.markdown(
         "<style>.line { border: 0.5px solid; margin: 0; }</style><div class='line'></div>",
         unsafe_allow_html=True)
+    
+    genre_selection = st.selectbox(
+        "Genre:",
+        options=["-All-", "Indie", "Action", "Casual", 
+                 "Adventure", "Simulation", "RPG", "Strategy",
+                 "Sports", "Racing", "Multiplayer"]
+    )
```
Comment (mehrdadhalali) [inline dashboard/Dashboard.py:107]: Nice addition!

**Review by Leon-JS02 — APPROVED** [review]
Reapproval. Merge conflict fixed.

**Review by mehrdadhalali — APPROVED** [review]

**Pull request merged**